### PR TITLE
drivers/video: passthrough unknown ioctl commands for customized scenarios in fb driver

### PR DESCRIPTION
**Pull request by terry0012**
## Summary
passthrough the custom ioctl commands to framebuffer driver 

## Impact
None

## Testing
Daily

### CODE REVIEW

**Review by hartmannathan — COMMENTED** [review]
LGTM

File: drivers/video/fb.c [hunk 1]
```diff
@@ -674,8 +674,15 @@ static int fb_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
         break;
 
       default:
-        gerr("ERROR: Unsupported IOCTL command: %d\n", cmd);
-        ret = -ENOTTY;
+        if (fb->vtable->ioctl)
```
Comment (gustavonihei) [inline drivers/video/fb.c:677]:
```suggestion
        if (fb->vtable->ioctl != NULL)
```
nit: Make the pointer validation more explicit by comparing to `NULL`, following the pattern from above cases.

**Review by jerpelea — APPROVED** [review]
please add a commit message describing the change

File: drivers/video/fb.c (hunk 1 above)
Comment (terry0012) [reply drivers/video/fb.c]: Done

**Comment by terry0012** [comment]
> 
“drivers/video: passthrough unknown ioctl cmd to fb driver” is the commit message, should I add it to another place ?

**Comment by acassis** [comment]
Hi @terry0012 please include a comment in the commit message explaining why it is necessary

**Comment by terry0012** [comment]
> Hi @terry0012 please include a comment in the commit message explaining why it is necessary

got it.

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**